tests/block_tools.py

```diff
@@ -2031,6 +2052,9 @@ def create_block_tools(
     const_dict=None,
     keychain: Optional[Keychain] = None,
 ) -> BlockTools:
+    global create_block_tools_count
```

Comment:
why do we need these globals?

Reply:
We don't! I just want to see them go to 1 before we remove these two functions entirely.

Reply:
They allow us to print how many times the BlockTools create functions have been called.
They aren't going to stay.